crates/types/src/bounding_box.rs

```diff
@@ -0,0 +1,26 @@
+use coordinate_systems::Pixel;
+use geometry::rectangle::Rectangle;
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, SerializeHierarchy)]
+pub struct BoundingBox {
+    pub bounding_box: Rectangle<Pixel>,
```

Comment:
This may be generic over the Frame
```suggestion
pub struct BoundingBox<Frame> {
    pub bounding_box: Rectangle<Frame>,
```

Reply:
YAGNI